File: 3th.rb
class GoodDog
    def initialize(name)
      @name = name
    end
  
    def get_name
      @name
    end
  
    def speak
      "#{@name} says Whoop!"
    end
end
  
  sparky = GoodDog.new("Sparky")
  sparky.get_name
  sparky.speak

File: 3th.js
class GoodDog {
    constructor(name) {
      this._name = name
    };
  
    get get_name() {
      return this._name
    };
  
    get speak() {
      return `${this._name} says Whoop!`
    }
};
  
  let sparky = new GoodDog("Sparky");
  console.log(sparky.get_name);
  console.log(sparky.speak);